Documentation/supermuc.rst

```diff
@@ -196,3 +125,170 @@ Many post-processing scripts of SeisSol require Python dependencies.
 We describe how to use pip on SuperMUC at see :ref:`pypi_behind_firewall`.
 
 
+Using Sanitizer
+---------------
+
+Note that to use sanitizer (https://en.wikipedia.org/wiki/AddressSanitizer), SeisSol needs to be compiled with gcc.
```

Comment:
It's also possible to use clang, iirc; but one of the arguments doesn't work then (the `static-libasan` iirc)